modules/snooker/src/test/ts/atomic/operate/TransformOperationsTest.ts

```diff
@@ -1,86 +1,117 @@
 import { assert, UnitTest } from '@ephox/bedrock-client';
-import { Arr, Fun } from '@ephox/katamari';
-import { SugarElement } from '@ephox/sugar';
+import { Arr } from '@ephox/katamari';
+import { SugarElement, SugarNode } from '@ephox/sugar';
 import { Generators } from 'ephox/snooker/api/Generators';
 import * as Structs from 'ephox/snooker/api/Structs';
 import * as TransformOperations from 'ephox/snooker/operate/TransformOperations';
+import * as MockStructs from 'ephox/snooker/test/MockStructs';
 import TestGenerator from 'ephox/snooker/test/TestGenerator';
 
 UnitTest.test('TransformOperationsTest', () => {
-  const en = (fakeElement: any, isNew: boolean) => Structs.elementnew(fakeElement as SugarElement, isNew);
+  let originalElements: Structs.ElementNew[] = [];
+  let expectedElements: Structs.ElementNew[] = [];
+
+  const clearElements = () => {
+    originalElements = [];
+    expectedElements = [];
+  };
+
+  // Original
+  const enO = (text: string, isNew: boolean, elemType: keyof HTMLElementTagNameMap = 'td') =>
```

Comment:
do you need `keyof HTMLElementTagNameMap` or could you just say `'td' | 'th' | 'col'`